hack/update-payload-crds.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+crd_globs="\
+    config/v1/*_config-operator_*.crd.yaml\
+    quota/v1/*.crd.yaml\
+    security/v1/*.crd.yaml\
+    securityinternal/v1/*.crd.yaml\
+    authorization/v1/*.crd.yaml\
+    operator/v1alpha1/0000_10_config-operator_01_imagecontentsourcepolicy.crd.yaml
+    "
+
+# To allow the crd_globs to be sourced in the verify script,
+# wrap the copy action to prevent it running when sourced.
+if [ "$0" = "$BASH_SOURCE" ] ; then
+    for glob in ${crd_globs}; do
+        for f in ${SCRIPT_ROOT}/${glob}; do
+            cp $f ${SCRIPT_ROOT}/payload-manifests/crds/
+        done
```

Comment:
The `glob` variable is confusingly named, and these `for` loops are not doing what you might think they are doing.  Because it is not quoted, `${crd_globs}` undergoes parameter expansion, word splitting, _and_ pathname expansion, so `glob` is actually assigned each expanded file name, not a glob.  Then the inner `for` loop iterates over only 1 value for each iteration of the outer loop.  You could replace the inner `for` loop with a single `cp` command.  Also, it would be safer to quote `$f` and `$SCRIPT_ROOT`:
```suggestion
    for f in ${crd_globs}; do
        cp "$f" "${SCRIPT_ROOT}/payload-manifests/crds/"
    done
```